src/index.js

```diff
@@ -66,103 +45,64 @@ export default {
         return this._getDeviceFromDetails(browserDetails) !== null;
     },
 
-    // Extra methods
     async resizeWindow (/* id, width, height, currentWidth, currentHeight */) {
-        this.reportWarning('The window resize functionality is not supported by the "fbsimctl" browser provider.');
+        this.reportWarning('The window resize functionality is not supported by the "ios" browser provider.');
     },
 
     async takeScreenshot (id, screenshotPath) {
-        var command = 'xcrun simctl io ' + this.currentBrowsers[id].udid + ' screenshot \'' + screenshotPath + '\'';
+        var command = `xcrun simctl io ${this.currentBrowsers[id].udid} screenshot \'${screenshotPath}\'`;
 
-        childProcess.execSync(command);
+        childProcess.execSync(command, { stdio: 'ignore' });

```

Comment:
I'm guessing this is what made the verbose, ugly logs.

Reply:
Yup. `idb_companion` is pretty chatty to `stderr`. But if you're referring to this output:
```
 Warnings (1):
 --
  Was unable to take a screenshot due to an error.

  Error: Unable to locate the page area in the browser window screenshot at
  /Users/michaelhan/code/saucelabs/sauce-testcafe-runner/screenshots/2021-03-31_10-11-39/test-3/Safari_13.0.4_iOS_13.3/1.png,
  because the page area mark with ID 158711479 is not found in the
  screenshot.
      at calculateClipInfo
  (/Users/michaelhan/code/saucelabs/sauce-testcafe-runner/node_modules/testcafe/src/screenshots/crop.js:119:19)
      at Object.cropScreenshot
  (/Users/michaelhan/code/saucelabs/sauce-testcafe-runner/node_modules/testcafe/src/screenshots/crop.js:136:18)
      at async_queue_1.addToQueue
  (/Users/michaelhan/code/saucelabs/sauce-testcafe-runner/node_modules/testcafe/src/screenshots/capturer.js:146:40)
      at process._tickCallback (internal/process/next_tick.js:68:7)
passed
```

That's actually a legit testcafe warning. There's some screenshot post-processing that's applied to remove the window chrome, but they haven't implemented it for mobile browsers. I don't see anything in the plugin interface to disable it either, unfortunately. Even more annoying is testcafe prints the warning to `stdout` so we can't  suppress it with output redirection.